components/python/automat/manifests/sample-manifest.p5m

```diff
@@ -25,6 +25,8 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file path=usr/bin/automat-visualize
 file path=usr/bin/automat-visualize-2.7
 file path=usr/bin/automat-visualize-3.5
```

Comment:
Something looks wrong here. You have removed Python-2.7 support so I would expect all related files to be removed from the sample-manifest, too.

Reply:
Probably I forgot to clobber... damn I hate that we do not clean the prototype before each install.

Reply:
FIxed.